### ISSUE TITLE
@b4s4nt - WP Development

### ISSUE THREAD

**Issue by Gizmotronn** [acord-robotics/portal#48]
Basically we need a WP site that can support:

a) community forums (through buddypress & bbpress)
b) gamification
c) integration with Disqus, Github, Reddit & Slack
d) Ideally I'd like a single sign on that will allow a user to log into github, disqus, slack and the wp site (and possibly Reddit)

Another software that we're looking at is Humhub, which is another Open-Source web software, that is used to build social networks. Once we get started with WP, I'll steer you/the team over to Humhub as well, so that we can see if Humhub is a viable solution for local hosting and whatnot. More details are coming soon :)

Basically we need to create a new WP theme (similar to Olympus or Buddyboss), or at the very least edit those themes. If you don't know anyone that does WP dev and you are only starting out learning (like I was last year), just getting familiar with WP and learning how to develop simple stuff would be good
